pkg/cmd/pr/shared/finder_test.go

```diff
@@ -341,27 +416,24 @@ func TestFind(t *testing.T) {
 					}}}`))
 			},
 			wantPR:   13,
-			wantRepo: "https://github.com/OWNER/REPO",
+			wantRepo: "https://github.com/ORIGINOWNER/REPO",
 		},
 		{
-			name: "current branch with upstream RemoteURL configuration",
+			name: "the current branch is configured to push to and pull from a URL (upstream, in this example) that is different from what the repo is configured to push to and pull from (origin, in this example) and push.default = upstream, it finds the PR associated with the upstream repo and returns origin as the base repo",
```

Comment:
Could we capture more of this verbose clarification as comments while keeping the test name a little more concise?

I think the information is valuable but not in the error message raised if/when the test case fails.